Info around open-source projects not officially supported by Segment (#6535)
segmentio/segment-docs: src/guides/how-to-guides/import-historical-data.md

### Demo projects

> info ""

> > The following projects are open-source and not officially supported by Segment. Should you face any issues, the best way to get help is by opening an issue on the project's GitHub page. 

Reviewer: ```suggestion
> The following projects are open-source and do not have official Segment support. If you encounter issues, the best way to get help is by opening an issue on the project's GitHub page. 
```

> > Feel free to clone the repository and adjust the code to suit your unique needs!

Reviewer: ```suggestion
> Feel free to clone the repository and adjust the code to suit your unique needs.
```
